### ISSUE TITLE
Text input on Q platform views results in log spam

### ISSUE THREAD

**Issue by mklim** [flutter/flutter#40716]
The workaround for #38375 results in a huge amount of "error" logs in adb:

`E/InputMethodManager(15162): b/117267690: Failed to get fallback IMM with expected displayId=29 actual IMM#displayId=0 view=io.flutter.plugins.webviewflutter.InputAwareWebView`.

We should prevent this. We may need to patch Android to remove the verbose log.

**Comment by aboutmydreams** [comment]
some problem :
E/InputMethodManager( 9778): b/117267690: Failed to get fallback IMM with expected displayId=32 actual IMM#displayId=0 view=com.pichillilorenzo.flutter_inappwebview.InAppWebView.InAppWebView{5d3087a VFEDHVC.. ......ID 0,0-992,1913}

**Comment by zzzhr1990** [comment]
E/InputMethodManager(11577): b/117267690: Failed to get fallback IMM with expected displayId=172 actual IMM#displayId=0 view=io.flutter.plugins.webviewflutter.InputAwareWebView{9e5510a VFEDHVC.. .F...... 0,0-1080,2097}

same problem

**Comment by DomingoMG** [comment]
Failed to get fallback IMM with expected displayId=128 actual IMM#displayId=0 view=io.flutter.plugins.webviewflutter.InputAwareWebView{dd4a55c VFEDHVC.. .F...... 0,0-1080,1584}

**Comment by prabhu73** [comment]
@mklim The issue is still there on beta channel also. Please find the below details

**E/InputMethodManager: b/117267690: Failed to get fallback IMM with expected displayId=80 actual IMM#displayId=0 view=com.pichillilorenzo.flutter_inappwebview.InAppWebView.InAppWebView{7aab15 VFEDHVCL. .F...... 0,0-1080,990}**

I am using flutter_inappwebview library, when I try to load any URL in flutter webview, the above log is displaying and when I try to enter text on input field which is there in webview page, the complete screen becoming black color and the page is not responding for hardware back button.

```
flutter doctor terminal logs
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, 1.22.0-12.1.pre, on Mac OS X 10.15.6 19G2021, locale en-IN)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
[✓] Android Studio (version 4.0)
[✓] IntelliJ IDEA Community Edition (version 2020.1.2)
[✓] VS Code (version 1.47.1)
[✓] Connected device (1 available)

• No issues found!
```

**Comment by hunterxxx** [comment]
any solution?

**Comment by blasten** [comment]
This has been fixed in Flutter 1.22/ WebView 1.0 - use hybrid composition in the Flutter WebView plugin.

https://github.com/flutter/plugins/tree/master/packages/webview_flutter#using-hybrid-composition
